src/Illuminate/Foundation/Http/FormRequest.php

```diff
@@ -188,11 +195,19 @@ protected function failedAuthorization()
     /**
      * Get the validated data from the request.
      *
-     * @return array
+     * @param  string|null  $key
+     * @param  mixed|null  $default
+     * @return  mixed
+     *
+     * @throws \Illuminate\Validation\ValidationException
      */
-    public function validated()
+    public function validated(string $key = null, $default = null)
```

Comment:
This is a breaking change for anyone overwriting this method.

Reply:
True, maybe consider for Laravel 9 then 😅